src/ui-lib/sass/05-components/02-molecules/00-navigation/_00-nav-menu.scss

```diff
@@ -16,17 +17,14 @@
 
     @media (min-width: $trigger-breakpoint) {
       display: flex;
-      margin-top: 0;
-      text-align: right;
+      flex-wrap: wrap;
+      margin: 0 (-$grav-c-nav-menu-link-margin) 0 (-$grav-sp-s);
```

Comment:
Perhaps use `$grav-c-nav-menu-link-margin` for the left margin value as well (or , `$nav-link-padding` if you rename it as suggested above). I think that would make the purpose of the negative margin clearer.

Reply:
Also - I've just had an idea (sorry!) - we could move this out of the media query, so that the negative left & right margins always apply (thus always making the text of nav links flush with the edges of the nav menu).

Then, we could remove that oversized width in header's CSS which is basically trying to achieve the same thing.